src/styles/index.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import './utils/reset';
```

Comment:
Where are these `utils` you are exporting from located? I can't see this folder in your project.